### PR TITLE
JSON converter work

### PR DESCRIPTION
**Pull request by jouva**
This will add the pontojson script as well as format the documentation for the repo

### CODE REVIEW

File: pontojson.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    "Xtra": "symbol-extracredit.png",
+    "Adventure": "symbol-adventure.png",
+    "Custom": "symbol-custom.png",
+    "Power": "symbol-power.png",
+    "Multiplicity": "symbol-multiplicity.png",
+    "Canon": "symbol-canon.png",
+    "Dungeon": "symbol-dungeon.png",
+    "2014": "symbol-2014.png",
+    "2015": "symbol-2015.png",
+    "Workshop": "symbol-workshop.png",
```
Comment (CodesignDev) [inline pontojson.py]: Is this needed, as there hasn't been a Workshop symbol in any of the current sets, or is this for future stuff?

Comment (marisathewitch) [inline pontojson.py]: It's all current sets.
and i thibk all picture now avaliable

Comment (jouva) [inline pontojson.py]: No there's a bunch that were in the expansion icon list but had no file. Namely:
- Adventure Pack
- Custom Cards
- Dungeon Delvers
- 2014 Con Exclusives
- 2015 Con Exclusives
- Hearthswarming
- Gracious Givers
- Weeaboo Paradaisu

This WAS however missing the Panels of Ponies expansion icon reference

File: pontojson.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+"""
+pontojson.py: Convert Horrible People Games' cards.pon files to .json format
+For use with the TSSSF Card Generator module
+"""
+__author__ = "Jouva Moufette"
+__copyright__ = "Copyright 2016, Children of Kefentse"
+__credits__ = ["", ""]
```
Comment (jouva) [inline pontojson.py]: I don't know why this is still here

File: pontojson.py [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+__license__ = "CC-BY-SA"
+__version__ = "1.0.0"
+__maintainer__ = "Jouva Moufette"
+__email__ = "jouva@moufette.com"
```
Comment (jouva) [inline pontojson.py]: I should change this now

File: README.md [hunk 4]
```diff
@@ -33,29 +33,29 @@ The script will open the filepath to cards.pon (or whatever you name it), and wi
...
- * {race change} = "When you attach this card to the grid, you may choose one Pony card attached to this Ship. Until the end of your turn, that Pony card becomes a race of your choice. This cannot affect Changelings."
- * {timeline change} = "When you attach this card to the grid, you may choose one Pony card attached to this Ship. Until the end of your turn, that Pony card counts as {postapocalypse}."
- * {play from discard} = "You may choose to play the top card on the Pony discard pile with this Ship, rather than use a Pony card from your hand."
\ No newline at end of file
+* `{replace}` = "While in your hand, you may discard a Pony card from the grid and play this card in its place. This power cannot be copied."
+* `{swap}` = "You may swap 2 Pony cards on the shipping grid."
+* `{3swap}` = "You may swap up to 3 Pony cards on the grid."
+* `{draw}` = "You may draw a card from the Ship or Pony deck."
+* `{goal}` = "You may discard a Goal and draw a new one to replace it."
+* `{search}` = "You may search the Ship or Pony discard pile for a card of your choice and play it."
```
Comment (n-a-larson) [inline README.md]: These are not the latest wordings

**Comment by CodesignDev** [comment]
:+1:

File: README.md [hunk 5]
```diff
@@ -33,29 +33,30 @@ The script will open the filepath to cards.pon (or whatever you name it), and wi
...
- * {timeline change} = "When you attach this card to the grid, you may choose one Pony card attached to this Ship. Until the end of your turn, that Pony card counts as {postapocalypse}."
- * {play from discard} = "You may choose to play the top card on the Pony discard pile with this Ship, rather than use a Pony card from your hand."
\ No newline at end of file
+* `{replace}` = "While this card is in your hand, you may discard a Pony card from the grid and play this card in its place. This power cannot be copied."
+* `{swap}` = "You may swap 2 Pony cards on the shipping grid."
+* `{3swap}` = "You may swap up to 3 Pony cards on the grid."
+* `{draw}` = "You may draw 1 card from the Ship or Pony deck."
+* `{goal}` = "You may discard 1 active Goal and draw 1 new Goal to replace it."
+* `{search}` = "You may search the Ship or Pony discard pile for 1 card of your choice and put it into your hand. If it's still in your hand at the end of your turn, discard it."
+* `{copy}` = "You may copy the power of any Pony card currently on the shipping grid, except for Changelings."
```
Comment (n-a-larson) [inline README.md]: Replace "Changelings" with "Ponies with the Changeling keyword"

File: README.md [hunk 6]
```diff
@@ -33,29 +33,30 @@ The script will open the filepath to cards.pon (or whatever you name it), and wi
...
+* `{hermaphrodite}` = "May count as either `{male}` or `{female}` for all Goals, Ships, and powers."
+* `{double pony}` = "This card counts as 2 Ponies."
+* `{love poison}` = "Instead of playing this Ship with a Pony card from your hand, or connecting two Pony cards already on the grid, you may take a Pony card from the shipping grid and reattach it elsewhere with this Ship. That card's power activates."
+* `{keyword change}` = "When you attach this card to the grid, you may choose one Pony card attached to this Ship. Until the end of your turn, that Pony card gains one keyword of your choice, except for Pony names."
+* `{gender change}` = "When you attach this card to the grid, you may choose one Pony card attached to this Ship. Until the end of your turn, that Pony card becomes the opposite gender."
+* `{race change}` = "When you attach this card to the grid, you may choose one Pony card attached to this Ship. Until the end of your turn, that Pony card becomes the race of your choice. This cannot affect Changelings."
```
Comment (n-a-larson) [inline README.md]: Replace "Changelings" with "Ponies with the Changeling keyword"

File: README.md (hunk 5 above)
Comment (jouva) [inline README.md]: Can you change this in the game module as well?

File: README.md (hunk 6 above)
Comment (jouva) [inline README.md]: Can you change this in the game module as well?

File: README.md (hunk 5 above)
Comment (n-a-larson) [inline README.md]: do you mean the CardMachine?

**Comment by n-a-larson** [comment]
:+1: I verified the rules-wordings, and I reviewed the python as well as I could.

**Pull request merged**